examples/nextjs/package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "@evolv/nextjs-demo",
+  "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev --port 4000",
-    "clean": "del dist **/*.tsbuildinfo .next",
+    "dev": "PORT=4001 next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "eslint . --ext=.ts,.tsx"
+    "lint": "next lint"
   },
   "dependencies": {
-    "@evolv/nextjs": "*",
+    "@evolv/nextjs": "^1.0.7",
     "cookies-next": "^2.1.1",
-    "next": "latest",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "next": "12.3.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "18.11.2",
+    "@types/react": "18.0.21",
+    "@types/react-dom": "18.0.6",
+    "eslint": "8.25.0",
+    "eslint-config-next": "12.3.1",
+    "typescript": "4.8.4"
```

Comment:
These *technically* shouldn't be necessary. This repo uses workspaces, so each subproject will inherit dependencies from the root project.